internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -407,10 +407,7 @@ func TestAccClusterAdvancedClusterConfig_replicationSpecsAndShardUpdating(t *tes
 				Config: configMultiZoneWithShards(orgID, projectName, clusterName, 1, 1, false),
 				Check:  checkMultiZoneWithShards(clusterName, 1, 1),
 			},
-			{
-				Config: configMultiZoneWithShards(orgID, projectName, clusterName, 2, 1, false),
-				Check:  checkMultiZoneWithShards(clusterName, 2, 1),
-			},
+			// TODO: updating from single sharded to using old schema should throw an error here
```

Comment:
this second update is no longer working because the first config is compatible with the new schema (`num_shards` = 1) and is populating `external_id` (and not `id`). This is a case where we will throw an error if user tries to update going into the old schema structure (`num_shards` > 1)

Reply:
Can this be captured in the test?

Reply:
Yes, as part of https://jira.mongodb.org/browse/CLOUDP-259828 we will add the validation and assert the error in this test